CSC631-MultiplayerGameDev-Group-Project/TeamFourProject
# Feature UI main to quit

# About
Can transition from Main Menu's 'Quit' button to the 'Exit Application' scene.

# Testing this pull request
1. Activate the MainMenuScene
2. Play
3. Click 'Quit' button
4. Observe that the scene has changed to the 'Exit Application' scene
